bagdir: "/root/catkin_ws/src/Legged_bags/Bags/" # may have to create this directory
record_command_file: "/root/catkin_ws/src/dawg_core/rosbag_record_command.txt"
diagnostics_topic: "/SOC_diagnostics"
camera:
  enable: False
  width: 640  # Default width
  height: 480  # Default height
  fps: 30  # Default frames per second
  fov: 87.0 # default fov of d400 series
  pos: [0.15, 0.047, 0.02] ## use (0.15, 0.047, 0.02) for D455, (0.15, 0.025, 0.02) for d435
  dir: [0, -1, 0]
  up: [0, 0, 1]
  rot: [0, 0, 0, 1] ## technically, the up and forward dir should be derivable from this, however, I need to talk to the BnG devs to figure out the conversion from REP103 to BnG
  color_optical_frame: "camera_color_optical_frame"
  depth_optical_frame: "camera_depth_optical_frame"
  depth_frame: "camera_depth_frame"
  camera_color_topic: "/camera/color/image_raw"
  camera_depth_topic: "/camera/depth/image_rect_raw"
  camera_color_info_topic: "/camera/color/camera_info"
  camera_depth_info_topic: "/camera/depth/camera_info"
  monitor_topic: "/camera/depth/image_rect_raw" ## this is the topic HAL uses for monitoring purposes
  annotation: False
lidar:
  enable: False
  rays_per_second_per_scan: 5000  # Default points
  channels: 3  # Default height. in execution we cut the first and last channel if there are only 3 channels to simulate a 2D scanner
  fps: 10  # Default frames per second
  vertical_angle: 26.9 # this is only for simulation purposes
  pos: [0.04, 0, 0.07] # default position of lidar wrt imu
  rot: [0, 0, 0, 1]
  dir: [0, -1, 0]
  up: [0, 0, 1]
  frame: "laser_frame"
  max_distance: 10.0
  scan_topic: "/scan"
  monitor_topic: "/scan"
  pc_topic: "converted_pc"
vesc:
  fps: 50.0
  erpm_gain: 3166
  topic: "sensors/core"
mavros:
  pos: [0,0,0.15] # default position on the hound
  fps: 50
  monitor_topic: "/mavros/imu/data_raw"
  pose_topic: "/mavros/local_position/pose"
  odom_topic: "/mavros/local_position/odom"
  state_topic: "/mavros/state"
  gps_topic: "/mavros/gpsstatus/gps1/raw"
  notification_topic: "/mavros/play_tune"
  channel_topic: "mavros/rc/in"
  raw_input_topic: '/mavros/manual_control/send'
  frame: "base_link"
  failure_action: "rosrun mavros mavsys rate --all 50"
dawg:
  dawg_low_launch: "roslaunch unitree_legged_control real.launch ctrl_level:=lowlevel"
  dawg_high_launch : "roslaunch unitree_legged_control control_high.launch ctrl_level:=highlevel"
  dawg_walk : "roslaunch unitree_legged_control real_high_walk.launch ctrl_level:=highlevel"
